src/tests/lxc-test-unpriv

```diff
@@ -27,6 +27,42 @@ if [ $(id -u) -ne 0 ]; then
 	echo "ERROR: Must run as root."
 	exit 1
 fi
+
+# Test if we're using an overlayfs module that handles symlinks correctly. If
+# not, we skip these tests since overlay clones will not work correctly.
+modprobe -q overlayfs
+if [ "${?}" -eq 0 ]; then
```

Comment:
It just irritated me because in my mind, this line should only trigger when then command returns a non-zero exit status and not, as it actually does, when it returns a zero exit status. In my mind writing `if modprobe -q overlayfs; then` implies `if some command != 0; then` so that the `if` block is hit when `modprobe -q overlayfs` fails while we actually want to hit it when it succeeds. Calling before and explicitly comparing exit status seemed clearer to me. But I'm happy to simplify as well.